tests/test_utils_misc.py

```diff
@@ -18,6 +19,17 @@ def test_non_ascii_unicode(self):
     def test_non_ascii_bytes(self):
         assert get_crc32(u'example\u5000'.encode('utf8')) == 1259721235
 
+    def test_negative_crc32(self):
+        assert get_crc32(b'1') == -2082672713
+
+    def test_crc32_range(self):
```

Comment:
Is this test fails when get_crc32 used without to_signed32? Sorry, for being picky @lopuhin